### PR TITLE
Rename cmpt2 to cmpo several more places

### PR DESCRIPTION
**Pull request by jkingdon**
Some pull requests crossed in flight so this got missed the first time.

### CODE REVIEW

**Comment by jkingdon** [comment]
Thanks to @GinoGiotto for quickly noticing this and reporting it.

**Comment by jkingdon** [comment]
I'll wait for an approval (as usual) but feel free to merge if you see this before I do and have merge capabilities.

**Review by wlammen — APPROVED** [review]

**Pull request merged**
